index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main class="container">
+        <header class="notification">
+            <div class="logo"><img src="img/My Logo.png" alt="Logo"></div>
+            <div class="hamburger"><nav><img src="img/Union.png" alt="menu"></nav></div>
+        </header>
+        <div class="main_body">
+            <div class="header2">I'm Ali<br> Glad to meet you!</div>
```

Comment:
Kindly increase the font size and weight of your main title so it better matches the figma design:

Your design:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/37116322/219764388-120f4fb8-c917-47ac-a82f-36cca37cdd48.png">

Figma design:
<img width="163" alt="image" src="https://user-images.githubusercontent.com/37116322/219764512-ff24f956-7b7e-40ec-bce5-5ad4ac20f6cd.png">